src/ui/DiffView.cpp

```diff
@@ -804,7 +815,9 @@ class HunkWidget : public QFrame
     mHeader = new Header(diff, patch, index, lfs, submodule, this);
     layout->addWidget(mHeader);
 
-    mEditor = new Editor(this);
+    auto edt = new Editor(this);
```

Comment:
`auto*` for clarity

Reply:
Thanks for advice. I didn't know auto* can be used for pointers. Will change it.